app/views/transition_landing_page/_page_header.html.erb

```diff
@@ -11,7 +11,13 @@
   <meta name="twitter:card" content="summary_large_image">
   <meta name="twitter:image" content="<%= image_url("transition-period.png") %>">
   <meta name="description" content="<%= t("transition_landing_page.meta_description") %>">
```

Comment:
Should we convert these tags to the gem too?

Reply:
I asked about this in govuk frontenders as it would be nice and tidy. Here's aquote from Alex J on the subject:

>my understanding is that the reason for having a simple tag (meta that is) as a component is to ensure consistency and have control over the name attribute which is being used as a contract between the values (provided by the backend) and the script that picks these values up. based on this rationale I wouldn’t find much value in extending/rescoping the component to cover all meta tags (opengraph and such) – unless we have a plan to standardise the open graph tags across gov.uk, in which case we could consider it

With this in mind I think for now it makes sense to me to keep the meta tags component as purely a delivery means for `govuk` prefixed tags instead of trying to make it all encompassing.